calico_mesos/calico_mesos.py

```diff
@@ -601,6 +603,22 @@ class IsolatorException(Exception):
         _log.error(e)
         sys.stdout.write(_error_message(str(e)))
         sys.exit(1)
+    except DatastoreError as e:
+        # Encountered an etcd error
+        _log.error(e)
+        # Try to give a more helpful error message depending on whether or not
+        # the user has set ETCD_AUTHORITY
+        try:
+            etcd_authority = os.environ['ETCD_AUTHORITY']
+            error_message = ("Failed to communicate with etcd at '%s'. "
```

Comment:
You're mixing parens and \ a newline continuation.

Not sure how common it is to use parens to staple a string together over multiple lines.  I'd probably stick to using \ instead.  Same below.